opencog/rule-engine/Rule.h

```diff
@@ -334,3 +340,4 @@ std::string oc_to_string(const RuleTypedSubstitutionMap& rule_ts_map);
 } // ~namespace opencog
 
 #endif /* RULE_H_ */
+
```

Comment:
Hmm, is that your editor adding these? So if I may ask, what is this unholy editor of yours?